docs/src/main/asciidoc/security-enabling-basic-auth-howto.adoc

```diff
@@ -0,0 +1,22 @@
+[id="security-enabling-basic-auth-howto"]
+= Enable HTTP Basic authentication
+
+Enable the HTTP Basic authentication for your Quarkus project and allow users to authenticate with a username and password.
+ 
+== Prerequisites
+ 
+* You have installed at least one extension that provides an `IdentityProvider` based on username and password, such as xref:security-jdbc.adoc[Elytron JDBC].
+ 
+. Enable Basic authentication by setting the value of `quarkus.http.auth.basic` property to `true`.
++
+[source,properties]
+----
+quarkus.http.auth.basic=true
+----
+ 
+For a Basic authentication configuration walk-through that uses `JPA`, see the xref:security-getting-started.adoc[Getting Started With Security] guide.
+ 
+== Additional resources
+ 
```

Comment:
@hmanwani-rh Hi Heena, thanks what I'd like to retain is for every security document link back to a parent `security.adoc`, a user should be able to get to back to it from any of the security related resources. 
I see you linking to an individual section there which makes sense, but having a dedicated link to a parent `security.adoc` should be good IMHO

Reply:
Done, added cross-reference in additional resources for security.adoc.